packages/block-editor/src/components/block-toolbar/style.scss

```
@@ -44,6 +44,16 @@
 	}
 }
 
+.block-editor-block-contextual-toolbar.with-offset {
```

To match the previous suggestion:
```suggestion
.block-editor-block-contextual-toolbar.has-offset {
```